scripts/run_validator.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
-tag=latest
+tag=add-support-for-address
```

Comment:
Reminder to revert before merge
```suggestion
tag=latest
```